mappings/net/minecraft/block/WallBlock.mapping

```diff
@@ -1,19 +1,34 @@
 CLASS net/minecraft/unmapped/C_duybclay net/minecraft/block/WallBlock
+	FIELD f_asmqtyre WALL_COVER_END I
 	FIELD f_bagxnabu WEST_SHAPE Lnet/minecraft/unmapped/C_cgckxfsw;
+	FIELD f_euglojbj WALL_COVER_START I
 	FIELD f_iildrtsa WATERLOGGED Lnet/minecraft/unmapped/C_xhwijdsd;
+	FIELD f_jsqywopd POST_COVER_WIDTH I
 	FIELD f_jvxfdifk SOUTH_SHAPE Lnet/minecraft/unmapped/C_cgckxfsw;
+	FIELD f_mfwaedwm POST_WIDTH I
 	FIELD f_mxflxjfq TALL_EAST_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_oihkjtbb collisionShapeMap Ljava/util/Map;
 	FIELD f_osozlsht TALL_WEST_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_pvverwec shapeMap Ljava/util/Map;
 	FIELD f_qkhypgkr EAST_SHAPE Lnet/minecraft/unmapped/C_cgckxfsw;
+	FIELD f_rmgxbzis WALL_WDITH I
 	FIELD f_rqtjcogh TALL_SOUTH_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_syithcwx TALL_NORTH_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_uqirvzwg TALL_POST_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_wcrdjxol NORTH_SHAPE Lnet/minecraft/unmapped/C_cgckxfsw;
 	FIELD f_wfwahvuy UP Lnet/minecraft/unmapped/C_xhwijdsd;
+	FIELD f_xdlnrllu WALL_HEIGHT I
+	METHOD m_huumgauz makeWallState (ZLnet/minecraft/unmapped/C_zscvhwbd;Lnet/minecraft/unmapped/C_zscvhwbd;)Lnet/minecraft/unmapped/C_seqcnhpn;
```

Comment:
```suggestion
	METHOD m_huumgauz getWallShape (ZLnet/minecraft/unmapped/C_zscvhwbd;Lnet/minecraft/unmapped/C_zscvhwbd;)Lnet/minecraft/unmapped/C_seqcnhpn;
```

This has nothing to do with a block state. Please be careful when porting mojang names.

Reply:
I see, I thought i change that name before i pushed, but clearly not. My bad